rgw/v2/tests/s3_swift/test_object_expiration.py

```diff
@@ -0,0 +1,144 @@
+# test s3 bucket_lifecycle operations
+import os, sys
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+from v2.lib.resource_op import Config
+import v2.lib.resource_op as s3lib
+from v2.lib.s3.auth import Auth
+import v2.utils.log as log
+import v2.utils.utils as utils
+from v2.utils.utils import HttpResponseParser, RGWService
+import traceback
+import argparse
+import yaml
+import json
+import random, time
+from v2.lib.exceptions import TestExecError, RGWBaseException
+from v2.lib.rgw_config_opts import CephConfOp, ConfigOpts
+from v2.utils.test_desc import AddTestInfo
+from v2.lib.s3.write_io_info import IOInfoInitialize, BasicIOInfoStructure, BucketIoInfo
+from v2.tests.s3_swift import resuables
+from v2.lib.s3 import lifecycle as lc
+from v2.lib.s3 import lifecycle_ops as lc_ops
+
+TEST_DATA_PATH = None
+
+
+def test_exec(config):
+    io_info_initialize = IOInfoInitialize()
+    basic_io_structure = BasicIOInfoStructure()
+    write_bucket_io_info = BucketIoInfo()
+    io_info_initialize.initialize(basic_io_structure.initial())
+    ceph_conf = CephConfOp()
+    rgw_service = RGWService()
+    #import pdb;
+    #pdb.set_trace()
+    config.rgw_lc_debug_interval = 30
+    config.rgw_lc_max_worker = 10
+    log.info('making changes to ceph.conf')
+    ceph_conf.set_to_ceph_conf('global', ConfigOpts.rgw_lc_debug_interval, str(config.rgw_lc_debug_interval))
+    ori = utils.exec_shell_cmd("ceph version")
```

Comment:
`ori` should be more meaning full. use ceph_version or better naming convention

Reply:
re-named ori to ceph_version.